### DISCUSSION TITLE
Searchability across instances

### DISCUSSION THREAD

**Discussion by dorsiflexion** [mastodon/mastodon#28674] [Q&A] [answered]
Hello,
how does Mastodon make sure the searchability of stauses stays the same across instances?

I'm trying to understand Mastodon better and just stumbled upon this question. Let's say I don't want my statuses to be included in the search results, so I set:
> [ ] Include public posts in search results

If I inspect my Account or any of my statuses via the API, I don't see a flag for this option. So if another instance that is not my home instance fetches any of my statuses, how do they know I don't want my statuses to be searchable?

Thanks and regards

**Comment by renchap** [comment, marked as answer]
This is a flag on your account, not on individual statuses. Each instance checks if your accounts has the `indexable` flag, and if yes, it will index your statuses.

**Comment by dorsiflexion** [comment]
Thanks. Yes, I also just noticed that servers communicate via ActivityPub, of course, and not via the Mastodon API. In the ActivityPub representation of an account, the `indexable` flag is present. One can check this via: `curl -sH 'Accept: application/ld+json; profile="https://www.w3.org/ns/activitystreams"' "https://$INSTANCE/users/$USER"`.
